src/nvim/highlight_group.c

```diff
@@ -215,6 +215,7 @@ static const char *highlight_init_both[] = {
   "default link LspReferenceRead            LspReferenceText",
   "default link LspReferenceText            Visual",
   "default link LspReferenceWrite           LspReferenceText",
+  "default link LspHoverRange               LspReferenceText",
```

Comment:
Would `LspHoverTarget` be slightly more generic / meaningful for potentially other cases? The "range" part seems over-specific.

Maybe even something more generic like `LspReferenceTarget`

Reply:
Nice, yeah I'll do that